cmd/vgmanager/vgmanager.go

```diff
@@ -335,3 +352,10 @@ func readyCheck(mgr manager.Manager) healthz.Checker {
 		return nil
 	}
 }
+
+func newGRPCServer() *grpc.Server {
+	return grpc.NewServer(grpc.UnaryInterceptor(ErrorLoggingInterceptor),
+		grpc.SharedWriteBuffer(true),
+		grpc.MaxConcurrentStreams(2),
```

Comment:
Could you please add a comment on why we're setting MaxConcurrentStreams and NumStreamWorkers to 2?